pam/integration-tests/testdata/TestCLIIntegration/golden/authenticate_user_with_mfa

```diff
@@ -329,8 +329,8 @@ Acct mgmt return: <nil>
 
 ────────────────────────────────────────────────────────────────────────────────
 > ./pam_authd socket=/tmp/pam-cli-tests.sock
-Auth return: <nil>
-Acct mgmt return: <nil>
+Auth user 'user-mfa' success
```

Comment:
Do we know where all those golden files were broken and was sign off to merge?

Reply:
I don't understand what you mean by "broken". Those messages are printed by the PAM `main()` function, which is only built and used for the CLI tests (so we can see that our module returned the expected output/error). The actual module itself does not print anything.

Reply:
Well, if you look at the diff, it was returning nil/nil and now, it’s returning some success value. So, does this mean that before nil/nil was success and now we always print something, or was there any issue we didn’t spot previously when merging a previous PR which was changed from success to nil?

I want to ensure we haven’t missed anything by mistake.

Reply:
Yeah, everything is fine, it was just a change in the return results presentation, not in the actual return values.